test/integration/scripting_spec.mjs

```diff
@@ -2260,23 +2260,27 @@ describe("Interaction", () => {
           let storage = await getAnnotationStorage(page);
           expect(storage)
             .withContext(`In ${browserName}`)
-            .toEqual({
-              "22R": { value: false },
-              "25R": { value: true },
-              "28R": { value: false },
-            });
+            .toEqual(
+              jasmine.objectContaining({
```

Comment:
Why are these changes necessary?
To avoid future surprises, can't we still *explicitly* check for the expected values here?